### PR TITLE
Update trivia

### PR DESCRIPTION
**Pull request by andrewwinke123**
Good evening Hunter. I've got a pull request for the Edit/Update trivia feature for the front end.

### CODE REVIEW

File: src/pages/EditTrivia/EditTrivia.jsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  }
+
+  const handleSubmit = (evt) => {
+    evt.preventDefault()
+    props.handleUpdateTrivia(formData)
+  }
+
+  return (
+    <main>
+            <form onSubmit={handleSubmit}>
```
Comment (HeyThatsNeat) [inline src/pages/EditTrivia/EditTrivia.jsx:24]: indentation is wrong on line 24

File: src/services/triviaService.js [hunk 2]
```diff
@@ -42,8 +42,26 @@ async function showTrivia(triviaId) {
...
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify(triviaFormData)
+  })
+  return res.json()
+} catch (error) {
+  console.log(error)
+}
+}
+
```
Comment (HeyThatsNeat) [inline src/services/triviaService.js:60]: indentation is also off in lines 45-60.

**Review by HeyThatsNeat — CHANGES_REQUESTED** [review]
there are some indentation problems that should be looked at

File: src/pages/EditTrivia/EditTrivia.jsx (hunk 1 above)
Comment (andrewwinke123) [reply src/pages/EditTrivia/EditTrivia.jsx:24]: done

File: src/services/triviaService.js (hunk 2 above)
Comment (andrewwinke123) [reply src/services/triviaService.js:60]: and done

**Pull request merged**